test-app/package.json

```diff
@@ -7,8 +7,11 @@
     "node": ">= 5.0"
   },
   "devDependencies": {
+    "@types/node": "^7.0.0",
+    "@types/react": "^15.0.0",
     "@webpack-blocks/babel6": "^0.x",
     "@webpack-blocks/css-modules": "^0.x",
+    "@webpack-blocks/typescript": "^0.x",
```

Comment:
Please change to a proper version number. I already changed it for `babel6` and `css-modules` in the release branch :)